### ISSUE TITLE
Build debian Matrix.org packages for aarch64

### ISSUE THREAD

**Issue by szclsya** [element-hq/synapse#17212]
**Description:** 

Currently matrix.org's debian repository only contains packages for amd64 and i386. It would be useful for Raspberry Pi and aarch64 vps users (i.e Amphere Altra) if such repository also contains packages for aarch64.

Previously (https://github.com/matrix-org/synapse/issues/5685) it seems possible to use `deb-src` to work around the limitation but this method no longer works.

**Comment by erikjohnston** [comment]
I don't think we have anything *against* building for aarch64, but unfortunately it is quite low on our priority list.

**Comment by ZicPL** [comment]
I think RISC-V 64-bit architecture should be on that list, even if it was low priority. Element (matrix.org) could participate in solidarity for another open source project✌️
